draft-fv-rats-ear.md

```diff
@@ -249,41 +249,62 @@ result that doesn't carry a trustworthiness vector.
 
 EAR provides core semantics for describing the result of appraising attestation
 evidence.
-However, a given deployment may offer extra functionality to its relying
-parties, or tailor the attestation result to the needs of an application (e.g.,
+However, a given application may offer extra functionality to its relying
+parties, or tailor the attestation result to the needs of the application (e.g.,
 TEEP {{-teep}}).
 To accommodate such cases, the EAR claims-set can be extended by plugging new
 claims into the `$$ear-extension` CDDL socket.
 
 The rules that govern extensibility of EAR are those defined in {{-cwt}} and
 {{-jwt}} for CWTs and JWTs respectively.
-An up-to-date view of the registered claims can be obtained via the
-{{IANA.cwt}} and {{IANA.jwt}} registries.
 
-A deployment-specific extension will normally mint its claim from the "private
+An extension MUST NOT change the semantics of the base EAR claims-set.
+
+A receiver MUST ignore any unknown claim.
+
+## Unregistered claims
+
+An application-specific extension will normally mint its claim from the "private
 space" - using integer values less than -65536 for CWT, and Public or
 Private Claim Names as defined in {{Sections 4.2 and 4.3 of -jwt}} when
 serializing to JWT.
+
 It is RECOMMENDED that JWT EARs use Collision-Resistant Public Claim Names
 ({{Section 2 of -jwt}}) rather than Private Claim Names.
 
-If there is even the slightest chance that an application-specific extension
-will be used across multiple environments, the associated extension claim
+## Registered claims {#sec-registered-claims}
+
+If an extension will be used across multiple applications, or is intended to be
+used across multiple environments, the associated extension claims
 SHOULD be registered in one, or both, the CWT and JWT claim registries.
-Since there is in general no guarantee that an application will be confined
-within an environment, it is RECOMMENDED that application-specific extension
-claims are always registered.
 
 In general, if the registration policy requires an accompanying specification
 document (as it is the case for "specification required" and "standards
 action"), such document SHOULD explicitly say that the extension is expected to
 be used in EAR claims-sets identified by this profile.
 
-An extension MUST NOT change the semantics of the base EAR claims-set.
+An up-to-date view of the registered claims can be obtained via the
+{{IANA.cwt}} and {{IANA.jwt}} registries.
 
-A receiver MUST ignore any unknown claim.
+## Choosing between registered and unregistered claims
+
+If an extension is to support functionality of a specific application (e.g.
+Veraison Services), its claims SHOULD be unregistered.
```

Comment:
I wouldn't make this a prescription.  If the application wants to register, they can -- in fact, it's probably better, tough non-zero cost.

We could probably use a mild encouragement: _"[...] its claims MAY be registered"_